### ISSUE TITLE
Update SDM lesson (011) to CMIP6

### ISSUE THREAD

**Issue by jcoliver** [jcoliver/quick-r#12]
CMIP5 projections are outdated. The lesson uses `raster::getData()` to retrieve forecast data, but it doesn't look like the function has been updated to grab CMIP6 data. As a stop gap, could download data from https://www.worldclim.org/data/cmip6/cmip6_clim2.5m.html, use `raster::brick(x = "path/to/cmip6/data)` to load in the file and go from there (updating names of layers in RasterBrick with names from `bioclim.data`).

**Comment by jcoliver** [comment]
Note it is a bit more complex, because contemporary bioclimate variables are in a different scale than the CMIP6 data. Specifically, historical temperature data are in 10 x degrees Celsius. For reference, these are:

`temp_biovars <- c("bio1", "bio2", "bio4", "bio5", "bio6", "bio7", "bio8", "bio9", "bio10", "bio11")`

Will probably just want to fix this early on with the historical bioclimate data. by multiplying the appropriate layers by 0.1

```
temp_biovars <- c("bio1", "bio2", "bio4", "bio5", "bio6", "bio7", "bio8", "bio9", "bio10", "bio11")
for (biov in temp_biovars) {
  message("Updating ", biov)
  bioclim.data[[biov]] <- 0.1 * bioclim.data[[biov]]
}
```

**Comment by jcoliver** [comment]
Addressed in 36e6718. Note the forecast data were ultimately transformed for ease of lesson progression.
